shell/mixins/brand.js

```diff
@@ -23,13 +24,19 @@ export default {
         }
       });
     } catch (e) {}
+
+    // Setting this up front will remove `computed` churn, and we only care that we've initialised them
+    this.haveAppsAndSettings = !!this.apps && !!this.globalSettings;
   },
 
   data() {
-    return { apps: [], globalSettings: [] };
+    return {
+      apps: null, globalSettings: null, haveAppsAndSettings: null
```

Comment:
this is different from https://github.com/rancher/dashboard/pull/9092/files#diff-556137f68d06f23031944f124827daa23f6f1cb5c7c6fe4ade07260010874f9eR35 so I presume this is what we want?

Reply:
Covered in comment in original PR. These changes will go in to and supersede any tweaks in there.

Reply:
As per https://github.com/rancher/dashboard/pull/9269#discussion_r1261050249